src/engine/strat_engine/filesystem.rs

```diff
@@ -153,6 +153,21 @@ pub fn create_fs(dev_path: &Path) -> EngineResult<()> {
     Ok(())
 }
 
+pub fn grow_fs(dev_path: &Path) -> EngineResult<()> {
```

Comment:
They all appear to be calling out to other executables. Some of these like `mount` and `umount` are syscalls that we could probably call ourselves using nix. Others like `xfs_growfs` we will need until XFS devs give us an API we can wrap and call directly.